manifests/v/Velocidex/Velociraptor/0.6.3/Velocidex.Velociraptor.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Velocidex.Velociraptor
+PackageVersion: 0.6.3
```

Needs custom version matching
```suggestion
PackageVersion: 0.63.0
```